tests/installation/user_settings.pm

```diff
@@ -27,12 +40,13 @@ sub run {
         return;
     }
 
-    send_key 'alt-f';    # Select full name text field
-    wait_screen_change { type_string $realname };
-    send_key 'tab';      # Select password field
-    send_key 'tab';
-    $self->type_password_and_verification;
-    assert_screen 'inst-userinfostyped';
+    $self->enter_userinfo();
+    if (!check_screen 'inst-userinfostyped') {
+        record_soft_failure 'bsc#1041747';
+        diag "trying again with slow typing due to https://progress.opensuse.org/issues/46190\n";
+        $self->enter_userinfo(retry => 1);
```

Comment:
does this actually overwrite the previous content?

Reply:
as you can see in my verification run it does - since tabbing/selecting a field seems to also mark the old content